terraform/a2ha-terraform/modules/aws/main.tf

```diff
@@ -26,156 +22,21 @@ locals {
 }
 
 data "aws_subnet" "public" {                                  
-  count = length(var.private_custom_subnets) > 0 ? 3 : 0            
+  count = length(var.public_custom_subnets) > 0 ? 3 : 0            
   id    = local.public_subnet_ids_list[count.index]
 }
 
-data "aws_internet_gateway" "default" {
```

Comment:
why we removed this code ?

Reply:
add a dummy changes into this 
https://github.com/chef/automate/blob/main/components/automate-backend-deployment/README.md
to bump the packages automate-ha-deployment

Reply:
are we removing the vpc creation terraform ?

Reply:
We removed this because we are not creating NAT gateway. And aws_internet_gateway was only used in the creation of NAT gateway.